packages/components/src/toggle-group-control/toggle-group-control-option-base/component.tsx

```diff
@@ -142,19 +140,17 @@ function ToggleGroupControlOptionBase(
 				) : (
 					<Ariakit.Radio
 						disabled={ disabled }
-						render={
-							<button
-								type="button"
-								{ ...commonProps }
-								onFocus={ ( event ) => {
-									onFocusProp?.( event );
-									if ( event.defaultPrevented ) {
-										return;
-									}
```

Comment:
We used to check if the event was `defaultPrevented` or not - is this something we want to keep, and if not, why?

Reply:
I can't find a specific reason, I believe it was just to respect a consumer wanting to `preventDefault`. I don't think we need it, as you say, since we're not using `onFocus` anymore

Reply:
Thanks for confirming, folks 👍